packages/eslint-plugin/tests/rules/no-invalid-void-type.test.ts

```diff
@@ -419,6 +579,54 @@ ruleTester.run('allowInGenericTypeArguments: true', rule, {
         },
       ],
     },
+    {
+      code: 'type invalidVoidUnion = void | Map;',
+      errors: [
+        {
+          column: 25,
+          line: 1,
+          messageId: 'invalidVoidUnionConstituent',
+        },
+      ],
+    },
```

Comment:
This test is unrelated to the PR, but it adds coverage to part of the code that is currently uncovered (it seemed small enough to be part of the PR, even if it's unrelated).